spec/controllers/hyrax/generic_works_controller_json_spec.rb

```diff
@@ -4,7 +4,10 @@
   routes { Rails.application.routes }
 
   let(:user) { create(:user) }
-  before { sign_in user }
+
+  before do
```

Comment:
Oops. This change was from back when I had the `persist_regisered_roles` changes in the branch. Will remove.